src/lib/components/workflow/workflows-summary-configurable-table/table-empty-state.svelte

```diff
@@ -0,0 +1,83 @@
+<script>
+  import { page } from '$app/stores';
+  import Icon from '$lib/holocene/icon/icon.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { workflowError } from '$lib/stores/workflows';
+  import emptyImage from '$lib/vendor/empty-state-dark_2x.png';
+  import noResultsImages from '$lib/vendor/empty-state-light_2x.png';
+
+  $: query = $page.url.searchParams.get('query');
+
+  const samples = [
+    'samples-go',
+    'samples-java',
+    'samples-typescript',
+    'samples-python',
+    'samples-dotnet',
+    'samples-php',
+  ];
+</script>
+
+<div
+  class="flex flex-col lg:flex-row w-full h-auto rounded-bl-xl rounded-br-xl border-2 border-gray-900 overflow-hidden"
+>
+  <div
+    class="w-auto min-w-[280px] lg:min-w-[520px] p-8 flex flex-col gap-4 bg-white border-r-2 border-gray-900"
```

Comment:
```suggestion
    class="w-auto min-w-[280px] lg:min-w-[520px] p-8 flex flex-col gap-4 bg-white border-b-2 lg:border-b-0 lg:border-r-2 border-gray-900"
```
| Before | After |
|--|--|
|<img width="783" alt="Screenshot 2023-08-09 at 2 32 48 PM" src="https://github.com/temporalio/ui/assets/15069288/a2b1fab1-3c00-43b9-b448-736c015394fb">|<img width="788" alt="Screenshot 2023-08-09 at 2 33 35 PM" src="https://github.com/temporalio/ui/assets/15069288/03d52701-28c7-49eb-9674-92f1b1426f80">|